lib/algfld.gi

```diff
@@ -212,18 +212,38 @@ local f,p,nam,e,fam,colf;
 end;
 
 InstallMethod(AlgebraicExtension,"generic",true,
-  [IsField,IsUnivariatePolynomial],0,DoAlgebraicExt);
+  [IsField,IsUnivariatePolynomial],0,
+function(k,f) return DoAlgebraicExt(k,f,true);
+end);
+InstallMethod(AlgebraicExtensionNC,"generic",true,
+  [IsField,IsUnivariatePolynomial],0,
+function(k,f) return DoAlgebraicExt(k,f,false);
+end);
 
 RedispatchOnCondition(AlgebraicExtension,true,[IsField,IsRationalFunction],
   [IsField,IsUnivariatePolynomial],0);
 
+RedispatchOnCondition(AlgebraicExtensionNC,true,[IsField,IsRationalFunction],
+  [IsField,IsUnivariatePolynomial],0);
+
```

Comment:
Could you format this in the same way as below?

Reply:
I don't know what you mean here. I have just copied the existing lines and added the `NC`.

Reply:
I mean I like it better if it's formatted like this for reasons of line lengths (as it is done in lines 243 ff for AlgebraicExtension(NC))
```
RedispatchOnCondition(AlgebraicExtensionNC,true,
    [IsField,IsRationalFunction],
    [IsField,IsUnivariatePolynomial],0);
```